packages/standard-asset-apis/src/RoutedSender.ts

```diff
@@ -111,11 +110,12 @@ export class RoutedSender {
      * because this logic may vary depending on where/how it is being used
      */
     rejectRecord: (record: DataEntity, error: unknown) => void;
-
     constructor(
+        logger: Logger,
```

Comment:
we try to follow conventional practice when using and organizing arguments. We usually prioritize important arguments first and lead with accessory arguments at the end, so I would first have routes as that is the primary mapping, then its associated options, then the logger last as its mainly a debugging tool and not central to the business logic.